install.py

```diff
@@ -5,243 +5,287 @@
-
-    os.system(f"cp {WORKING_DIR}lib/{arch}/libncurses.so.6.4 {mojo_lib_path}/libncurses.so.6")
-    os.system(f"cp {WORKING_DIR}usr/lib/{arch}/libform.so.6.4 {mojo_lib_path}/libform.so.6")
-    os.system(f"cp {WORKING_DIR}usr/lib/{arch}/libpanel.so.6.4 {mojo_lib_path}/libpanel.so.6")
-    os.system(f"cp {WORKING_DIR}usr/lib/{arch}/libedit.so.2.0.70 {mojo_lib_path}/libedit.so.2")
-
-# install mojo
-mojo = shutil.which(f"PATH=$PATH:{home}/.modular/pkg/packages.modular.com_mojo/bin/ mojo") is not None
-if mojo:
-    print("Mojo is already installed... cleaning up")
-    os.system(f"PATH=$PATH:{home}/.modular/pkg/packages.modular.com_mojo/bin/ modular clean")
-
-os.system(f"LD_LIBRARY_PATH=$LD_LIBRARY_PATH:{mojo_lib_path} modular install mojo")
-
-# fix crashdb directory not found:
-os.makedirs(f"{home}/.modular/crashdb", exist_ok=True)
-
-
-def rc_path():
-    shell = param("SHELL")
-    if shell is not None:
-        match shell.split("/")[-1]:
-            case "bash":
-                return get_shell("bash", f"{param('HOME')}/.bashrc")
-            case "zsh":
-                return get_shell("zsh", f"{param('HOME')}/.zshrc")
-            case _:
-                path = input(
-                    "Please enter the path to your shell rc file (e.g. ~/.bashrc for bash) or press ENTER to skip:")
-                if path == "":
-                    return None
-                return path.replace("~", param("HOME"))
-
-    else:
-        return get_shell()
-
-
-def get_shell(found=None, file=None):
-    if found is not None:
-        yn = input(f"Found {found} shell, add exports to {file}? [y/N/other]")
-        yn = yn.lower()
-        if yn == "y":
-            return file
-        elif yn == "n":
-            return None
-        elif yn == "other" or yn == "o":
-            return get_shell()
-        elif yn == "":
-            print("Skipping...")
+
+        try:
+            os.makedirs(self.working_dir)
+        except OSError:
+            pass
+
+    def fedora_os(self) -> None:
+        url = "https://ftp.debian.org/debian/pool/main/n/ncurses/libtinfo6_6.4-4_amd64.deb"
+        if self.fedora:
+            subprocess.run("sudo dnf install binutils", shell=True)
+
+            urllib.request.urlretrieve(url, f"{self.working_dir}/libtinfo.deb")
+            subprocess.run(f"cd {self.working_dir} && ar -vx libtinfo.deb && tar -xf data.tar.xz", shell=True)
+            
+            if self.install_global:
+                shutil.copy(f"{self.working_dir}/lib/{self.arch}/libtinfo.so.6.4", "/usr/lib/")
+                os.symlink("/usr/lib/libtinfo.so.6.4", "/usr/lib/libtinfo.so.6")
+            else:
+                os.mkdir(f"{self.mojo_lib_path}")
+                shutil.copy(f"{self.working_dir}/lib/{self.arch}/libtinfo.so.6.4", f"{self.mojo_lib_path}/libtinfo.so.6")
+
+    def install_modular(self) -> None:
+        url = "https://raw.githubusercontent.com/sneekyfoxx/arch-mojo/seekyfoxx/PKGBUILD"
```

Comment:
I guess, it should be my repo here, so it's easier to update. Ah and it's actually the wrong PKGBUILD in the repo. I still need to delete this one. The right one is `aur/modular/PKGBUILD`

Reply:
Yea, I was testing something I should have undid the changes before I pushed it.